dockerfile/script/wildfly-cli/embed-server/configure-managed-executor.adoc

```diff
@@ -0,0 +1 @@
+|MANAGED_EXECUTOR_SERVICE_CORE_THREADS |managed-executor-service.core-thread |16 | Wildfly managed executor maximális száma (egyébként cpu szerint lenne dinamikus 2x)
```

Comment:
Magyar szöveg, melyik nyelven dokumentálunk? Én angol felé mennék, sok dolog inkább angolul van írva már most is.

Reply:
angol lesz, ez igy maradt meg, atirom ezeket is